WorkMode/index.md

```diff
@@ -114,7 +114,7 @@ Each repository, including the “core” WG repository, has a wiki instance. Me
 
 ## Patent Policy
 
-The WG's Charter defines the [Patent Policy for this group](https://www.w3.org/2017/04/publ-wg-charter/#patentpolicy):
+The WG's Charter defines the [Patent Policy for this group](https://www.w3.org/2018/03/jsonld-wg-charter.html#patentpolicy):
```

Comment:
This change actually results in a `300 Multiple Choices` response. The other URL worked fine. I may just change this back post-merge...unless you beat me to it. ;)

Reply:
Seems to work ok for me.  How did you get a 300 code?  We do want to point to the JSON-LD WG charter section vs the similar section in an unrelated WG charter.

Reply:
I do not understand the issue... there was clearly a bug in the original page (my mistake, apologies! and thanks @davidlehn), and https://www.w3.org/2018/03/jsonld-wg-charter.html#patentpolicy, or even https://www.w3.org/2018/03/jsonld-wg-charter#patentpolicy (ie, without the 'html' suffix) works for me (I tried it on three different browsers).

@BigBlueHat where is that 300 come from?